appinfo/app.php

```diff
@@ -3,3 +3,4 @@
 OC::$CLASSPATH['OC_User_SMB']='user_external/lib/smb.php';
 OC::$CLASSPATH['OC_User_FTP']='user_external/lib/ftp.php';
 OC::$CLASSPATH['OC_User_BasicAuth']='user_external/lib/basicauth.php';
+OC::$CLASSPATH['OC_User_XMPP']='user_external/lib/xmpp.php';
```

Comment:
@violoncelloCH we should discuss PSR4 and autoloading at some point. This isn't necessary anymore if the class files are named correctly :rocket:

Reply:
Oh, nice! Didn't know about that.
Yes, would be cool, if we could talk about that! Should I create an issue for this as a reminder?

Reply:
Sure! It's also documented at https://docs.nextcloud.com/server/stable/developer_manual/app/classloader.html#psr-4-autoloading